src/Microsoft.ML.Core/Data/VBuffer.cs

```diff
@@ -38,6 +38,17 @@ namespace Microsoft.ML.Runtime.Data
         /// </summary>
         public readonly int[] Indices;
 
+        /// <summary>
+        /// The explicitly represented values.
+        /// </summary>
+        public ReadOnlySpan<T> GetValues() => Values.AsSpan(0, Count);
+
+        /// <summary>
+        /// The indices. For a dense representation, this array is not used. For a sparse representation
+        /// it is parallel to values and specifies the logical indices for the corresponding values.
```

Comment:
Concreate example can make this great doc string further clearer.
```suggestion
        /// it is parallel to values and specifies the logical indices for the corresponding values.
        /// For example, if GetIndices returns [3, 5] and GetValues produces [98, 76], this value buffer stands for a 
        /// vector with (1) non-zeros values 98 and 76 respectively at the 4-th and 6-th coordinates and (2) and
        /// zeros on all other coordinates. 
```

Reply:
added.